Remove outdated bridge version in adapter

mialbu opened the pull request:
The bridge has been updated to v3 on both N3 testnet and mainnet. Hence, the code in the bridge adapter contract for bridge v2 is no longer needed.

I've added code to remove the concept of different bridge versions since there's currently only one active in N3 testnet and mainnet. Once there's a new version that has a different interface, the adapter needs to be updated anyway and the handling of different bridge versions can be handled in that new bridge adapter version.

mialbu committed:
feat: remove bridge version v2

OT-rebel commented on src/main/java/com/axlabs/neo/grantshares/GrantSharesBridgeAdapter.java:
```diff
@@ -327,8 +301,6 @@ public static void deploy(Object data, boolean update) {
             }
             Storage.put(context, WHITELISTED_FUNDER_KEY, whitelistedFunder);
-
-            Storage.put(context, BRIDGE_VERSION_KEY, 2);
```
since we no longer use a bridge version, can we also remove the 
```private static final int BRIDGE_VERSION_KEY = 0x10;```
?
and maybe use the value `0x10`  for storage removal ?

mialbu replied:
For the sake of good practice I prefer to keep the value there, so that we're completely aware of what storage keys are in use. If we use `0x10` inline and would decide to add a new storage value with a new key still in this v2 update, we'd need to be very careful with what key to use.

I can prefix it, e.g., `V1_BRIDGE_VERSION_KEY` to make it clear it's no longer needed once the contract is updated to v2.

OT-rebel replied:
yeah, this way is better!

mialbu replied:
Done in https://github.com/AxLabs/grantshares-contracts/pull/84/commits/e1d892efabce2f8d1f6b46ea2b2d6864513b1f76.

mialbu committed:
feat: prefix bridge version key with V1

The bridge version will be deleted from storage in the update to v2. Prefixing it with v1 makes clear it'll no longer be used in v2+.
